agents/mongodb/internal/profiler/aggregator/aggregator.go

```diff
@@ -99,6 +100,9 @@ func (a *Aggregator) Add(ctx context.Context, doc proto.SystemProfile) error {
 	// we had some activity so reset timer
 	a.t.Reset(a.d)
 
+	doc.Ns, _ = truncate.Query(doc.Ns)
```

Comment:
Any reason to call it here?

Reply:
From tests I found out that is needed here too. Because from Ns and Op it assembly Fingerprint and Example.